Assets/Mirror/Tests/Editor/WeaverAssembler.cs

```diff
@@ -1,14 +1,38 @@
+using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using System.Collections.Generic;
+using UnityEditor;
 using UnityEditor.Compilation;
 using UnityEngine;
 
 namespace Mirror.Tests
 {
     public class WeaverAssembler : MonoBehaviour
     {
-        public const string OutputDirectory = "Assets/Mirror/Tests/Editor/WeaverTests~/";
+        static string _outputDirectory;
```

Comment:
do we actually need to cache it?
how often is this function called?

Reply:
It is called multiple times per Weaver test